### DISCUSSION TITLE
Build-time image optimization

### DISCUSSION THREAD

**Discussion by sumedhdev** [vercel/next.js#24979] [Ideas]
I get that NextJS 10 does image optimization at request time to "reduce build times".

I don't see a reason to assume someone won't want to use NextJS for purely static sites. I'd recommend NextJS introduces _optional_ build-time image optimization for those users who don't need SSR and are not concerned with build times - especially if they're able to avoid the need for a lambda and just host from an S3 bucket. 

I'm sure NextJS can make something as awesome as Gatsby's build-time image optimization with features like traced SVG placeholders etc?

🤞

**Comment by FabulousCupcake** [comment]
Seems to be duplicate of 👉 https://github.com/vercel/next.js/discussions/19065
